packages/squiggle-lang/src/rescript/Distributions/PointSetDist/Continuous.res

```diff
@@ -278,6 +278,7 @@ let combineAlgebraicallyWithDiscrete = (
   op: Operation.convolutionOperation,
   t1: t,
   t2: PointSetTypes.discreteShape,
+  flip: bool,
```

Comment:
I think this is a bit sloppy. It should instead accept a named parameter, maybe with a default. 

```
t1: t,
t2: PointSetTypes.discreteShape,
~distreteIsNthArgument: [> first | second]
``` 

Maybe

Reply:
It is a bit sloppy... I think booleans is very smelly, indicating they should be split into two different functions. Original implementation had two different functions, but it wasn't immediately obvious how to remove the duplication... I like your solution than bool though